src/agent/agent.c

```diff
@@ -1890,6 +1890,8 @@ bool agent_start(Agent *agent) {
         int r = 0;
         sd_bus_error error = SD_BUS_ERROR_NULL;
 
+        hirte_log_infof("Starting hirte agent v%s", CONFIG_H_HIRTE_VERSION);
```

Comment:
I'd remove `v`, just `Starting hirte-agent 0.5.0` should be enough

Reply:
works for me